src/Pdf.php

```diff
@@ -234,6 +247,11 @@ public function getImageData(string $pathToImage, int $pageNumber): Imagick
 
         $this->imagick->readImage(sprintf('%s[%s]', $this->filename, $pageNumber - 1));
 
+        if ($this->backgroundColor !== null) {
```

Comment:
Can you change this to `if (! empty($this->backgroundColor)) {` to avoid situations where an empty string was passed?

Reply:
I have modified and pushed. Thanks for suggestion.